core/state/database.go

```diff
@@ -257,6 +259,13 @@ func (db *VerkleDB) OpenTrie(root common.Hash) (Trie, error) {
 	if err != nil {
 		panic(err)
 	}
+
+	if TestVKTOpenStateless {
+		r, err = verkle.ParseStatelessNode(payload, 0, root[:])
+		if err != nil {
+			panic(err)
+		}
+	}
```

Comment:
Getting this to work was interesting to understand the `stateless.go` implementation in go-verkle.
I think this is the correct way to do what I intend. (Look at the second bullet of the PR description)

Reply:
hmmm, if you need to open a stateless node, you are using an older, slower version. Please rebase and re-run the benchmarks.

Reply:
Oh no, there wasn't a need for it. I just remembered some old pprof image you shared and I remembered seeing `StatelessNodes` there, so wanted to also include a benchmark for that case.

If the replay-block benchmark isn't using a stateless VKT, I can remove that benchmark since isn't relevant anymore.
Is that the case?

(Actually, adding this stateless VKT benchmark was friction here, so glad if that isn't meaningful anymore)